starter-repos/app-template/config/envs/prod-central/deployment.yaml

```diff
@@ -0,0 +1,18 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: app
+spec:
+  template:
+    spec:
+      containers:
+      - name: app
+        env:
+        # Required for Skaffold to detect runtime language
+        # https://skaffold.dev/docs/workflows/debug/#go
+        - name: GOTRACEBACK
```

Comment:
This is only needed by dev environment. Staging and prod don't use skaffold.

Reply:
SG. Will update it.